Symfony/CS/Fixer/Contrib/PhpdocToCommentFixer.php

```diff
@@ -0,0 +1,196 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Fixer\Contrib;
+
+use Symfony\CS\AbstractFixer;
+use Symfony\CS\Tokenizer\Token;
+use Symfony\CS\Tokenizer\Tokens;
+
+/**
+ * @author Ceeram <ceeram@cakephp.org>
+ */
+class PhpdocToCommentFixer extends AbstractFixer
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function fix(\SplFileInfo $file, $content)
+    {
+        $tokens = Tokens::fromCode($content);
+
+        foreach ($tokens->findGivenKind(T_DOC_COMMENT) as $index => $token) {
+            $nextIndex = $tokens->getNextMeaningfulToken($index);
+
+            if (null === $nextIndex || $tokens[$nextIndex]->equals('}')) {
+                $token->override(array(T_COMMENT, '/*'.ltrim($token->getContent(), '/*'), $token->getLine()));
+                continue;
+            }
+
+            if ($this->isStructuralElement($tokens[$nextIndex])) {
+                continue;
+            }
+
+            if ($tokens[$nextIndex]->isGivenkind(T_FOREACH) && $this->isValidForeach($tokens, $index)) {
+                continue;
+            }
+
+            if ($tokens[$nextIndex]->isGivenkind(T_VARIABLE) && $this->isValidVariable($tokens, $index)) {
+                continue;
+            }
+
+            if ($tokens[$nextIndex]->isGivenkind(T_LIST) && $this->isValidList($tokens, $index)) {
+                continue;
+            }
+
+            $token->override(array(T_COMMENT, '/*'.ltrim($token->getContent(), '/*'), $token->getLine()));
+        }
+
+        return $tokens->generateCode();
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getDescription()
+    {
+        return 'Docblocks should only be used on structural elements.';
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getPriority()
+    {
+        /*
+         * Should be run before the PhpdocIndentFixer, PhpdocParamsFixer and NoEmptyLinesAfterPhpdocsFixer
+         * so that these fixers don't touch doc comments which are meant to be converted to regular comments
```

Comment:
So, is this mean that this should be the first phpdoc fixer of all ?
Asking in #887 context.